recipes/xhmm/meta.yaml

```diff
@@ -0,0 +1,22 @@
+package:
+  name: xhmm
+  version: 1.0
+
+build:
+  number 0
+  
+source:
+  git_url: https://bitbucket.org/statgen/xhmm.git
+  git_rev: cc14e52
+  
+requirements:
+  build:
+    - gcc
+    - lapack
+test:
+  commands:
+    - xhmm -h  2>&1 | grep Usage
+
+about:
+  home: https://atgu.mgh.harvard.edu/xhmm/index.shtml
+  license: "academic use only"
```

Comment:
Is there a license file in the source code? Then, you could specify it with ``license_file``.